evergreen/evergreen.yml

```diff
@@ -206,6 +206,7 @@ functions:
           TOPOLOGY=${TOPOLOGY} \
           AUTH=${AUTH} \
           SSL=${SSL} \
+          STORAGE_ENGINE=${STORAGE_ENGINE} \
```

Comment:
Note: I'm relying on others to review the Evergreen changes.

Reply:
me too